src/components/Homepage/Homeitems/Filter.js

```diff
@@ -0,0 +1,49 @@
+import { useDispatch } from 'react-redux';
+import { filterCasesByName } from '../../../redux/covidCases/cases';
+import world from './world.svg';
+
+const Filter = (data) => {
+  const dispatch = useDispatch();
+  const filterHandler = (e) => {
+    dispatch(filterCasesByName(e));
+  };
+  /* eslint-disable react/destructuring-assignment */
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.